registry/content.md

```diff
@@ -1,6 +1,8 @@
 # Docker Registry
```

Comment:
This probably should be changed to something else

I _think_ this was part of the reason for adding https://github.com/docker-library/docs/pull/2113/files#r801121627 (https://github.com/docker-library/docs/pull/1798), as users tended to confuse `Docker Registry` and `Docker Hub (Registry)`.

Reply:
Should it be simply just `Distribution`?

Reply:
Should we merge for now with this as-is, or do you want to resolve this before merge?

Reply:
let me rename it to `Distribution`, @tianon, since we now have this shiny new logo :)